### PR TITLE
feat : adding test driver

### PR DESCRIPTION
**Pull request by shakederin**

### CODE REVIEW

File: src/test-kit/instegram-clone-driver.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    this.commentsLocator = page.locator(".comment");
+  }
+
+  postComment = async (input: string) => {
+    await this.typeInput(input);
+    await this.page.keyboard.press("Enter");
+  };
+
+  typeInput = async (input: string) => {
+    await this.inputLocator.type(input);
```
Comment (vlad-karnauch) [inline src/test-kit/instegram-clone-driver.ts:26]: consider using `function` (everywhere)

File: src/test-kit/instegram-clone-driver.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  getComments = () => {
+    return this.commentsLocator;
+  };
+
+  getCommentsCount = async () => {
+    return await this.commentsLocator.count();
+  };
+
+  isInputRendered = async () => {
+    return await this.inputLocator.isVisible();
```
Comment (vlad-karnauch) [inline src/test-kit/instegram-clone-driver.ts:42]: you don't need to do `isVIsible()`, hidden elements also take space on the screen, you can use `waitFor()` to see if the element is on the page

**Review by vlad-karnauch — COMMENTED** [review]
my general advise is, if you don't know that you're going to use a function, don't add it

File: src/test-kit/instegram-clone-driver.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  private commentListLocator: Locator;
+  private formLocator: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.inputLocator = page.locator("#commentInput");
+    this.postCommentLocator = page.locator("#postComment");
+    this.commentListLocator = page.locator("#commentsList");
+    this.formLocator = page.locator("#inputForm");
+    this.commentsLocator = page.locator(".comment");
```
Comment (vlad-karnauch) [inline src/test-kit/instegram-clone-driver.ts:17]: condiser making these private, that way you can replace the ids

Comment (shakederin) [reply src/test-kit/instegram-clone-driver.ts:17]: what do you mean?
because lines 4-9 set all on private

**Pull request closed without merging**